lib/processors/bundlers/moduleBundler.js

```diff
@@ -35,8 +35,11 @@ const log = getLogger("builder:processors:bundlers:moduleBundler");
  *	<li>
  *		<code>require</code>: A 'require' section is transformed into a sequence of jQuery.sap.require calls. The
  *		list will be resolved like an include pattern list in any of the other sections and for each of the resolved
- *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.requireSync' is
+ *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.require' is
```

Comment:
Isn't this deprecated in UI5?
https://sapui5untested.int.sap.eu2.hana.ondemand.com/sapui5-sdk-internal/#/api/jQuery.sap%23methods/jQuery.sap.require

Reply:
Done